ruby-3.0.gemfile

```diff
@@ -53,8 +53,8 @@ gem 'webmock', '>= 3.10.0'
 gem 'webrick', '>= 1.7.0'
 
 group :check do
-  gem 'rbs', '~> 3.6', require: false
-  gem 'steep', '~> 1.7.0', require: false
+  gem 'rbs', '~> 3', require: false
+  gem 'steep', '~> 1', require: false
```

Comment:
That type of boundaries a basically - no boundaries, except more or equal version `1` (i.e why `~>`?)

```ruby
gem 'steep', '~> 1'
```

I would rather go with `> 1.7` to highlight that version 1.7 is no fit for us

```suggestion
  gem 'steep', '> 1.7', require: false
```

Reply:
Multiple constraints can be specified, I believe I was the one who typed the `~> 1.7` part and we actually need that minimum version for code in master, therefore I would suggest `~> 1.7, >= 1.8` as the constraint.

Reply:
I added multiple constraints, like @p-datadog suggested.